src/lib/constants.js

```diff
@@ -3,13 +3,22 @@ import { homedir } from 'os';
 /**
  * @typedef {'next' | 'svelte' | 'nuxt' | 'empty' | 'none'} UiType
  * @typedef {'sudoku' | 'tictactoe'} ExampleType
+ * @typedef {'single-node' | 'multi-node'} LightnetMode
+ * @typedef {'fast' | 'real'} LightnetType
+ * @typedef {'none' | 'full'} LightnetProofLevel
+ * @typedef {'rampup' | 'berkeley' | 'develop'} LightnetMinaBranch
  *
- * @type {{ uiTypes: UiType[], exampleTypes: ExampleType[], feePayerCacheDir: string }}
+ * @type {{ uiTypes: UiType[], exampleTypes: ExampleType[], feePayerCacheDir: string, lightnetWorkDir: string, lightnetModes: LightnetMode[], lightnetTypes: LightnetType[], lightnetProofLevels: LightnetProofLevel[], lightnetMinaBranches: LightnetMinaBranch[] }}
  */
 const Constants = Object.freeze({
   uiTypes: ['next', 'svelte', 'nuxt', 'empty', 'none'],
   exampleTypes: ['sudoku', 'tictactoe'],
   feePayerCacheDir: `${homedir()}/.cache/zkapp-cli/keys`,
```

Comment:
Mind using [path.resolve](https://nodejs.org/api/path.html#:~:text=The%20path.resolve()%20method,an%20absolute%20path%20is%20constructed.) here to handle this on Windows as well?

Reply:
Will use it with current changes but feePayer related paths will be updated as part of the work on https://github.com/o1-labs/zkapp-cli/issues/513.

Reply:
I thought the feePayer related paths were working correctly on windows. Either way, it will be easy to update the paths.